tree-sitter-uvl/src/scanner.cc

```diff
@@ -59,7 +59,7 @@ struct Scanner {
     int32_t first_comment_indent_lenght = -1;
     lexer->mark_end(lexer);
     auto skip = [&]() { lexer->advance(lexer, false); };
-    auto advance = [&]() { lexer->advance(lexer, true); };
+    // auto advance = [&]() { lexer->advance(lexer, true); };
```

Comment:
Was this not used before? I think we should just remove it then instead of the comment